<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="PDCDT01GY4ZVQ1" name="com.Amazon.Login1stscenario" tests="7" failures="1" timestamp="2019-12-26T12:09:18 GMT" time="14.378" errors="0">
  <testcase name="testLoginpageOpens" time="3.210" classname="com.Amazon.Login1stscenario"/>
  <testcase name="testUrlOpensPage" time="0.245" classname="com.Amazon.Login1stscenario"/>
  <testcase name="CheckCaptcha" time="0.408" classname="com.Amazon.Login1stscenario">
    <failure type="java.lang.AssertionError" message="expected [To better protect your account, please re-enter your password and then enter the characters as they are shown in the image below.] but found [Rs. 499.00 - Rs. 1,539.00from ₹ 499.00 to ₹ 1,539.00
Ends in 06:20:51]">
      <![CDATA[java.lang.AssertionError: expected [To better protect your account, please re-enter your password and then enter the characters as they are shown in the image below.] but found [Rs. 499.00 - Rs. 1,539.00from ₹ 499.00 to ₹ 1,539.00
Ends in 06:20:51]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.Amazon.Login1stscenario.CheckCaptcha(Login1stscenario.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CheckCaptcha -->
  <testcase name="checkCaptchaWorks" time="0.001" classname="com.Amazon.Login1stscenario">
    <skipped/>
  </testcase> <!-- checkCaptchaWorks -->
  <testcase name="enterPass" time="7.307" classname="com.Amazon.Login1stscenario"/>
  <testcase name="enterEmail" time="2.562" classname="com.Amazon.Login1stscenario"/>
  <testcase name="checkLogin" time="0.645" classname="com.Amazon.Login1stscenario"/>
</testsuite> <!-- com.Amazon.Login1stscenario -->
